src/ray/raylet/placement_group_resource_manager.h

```diff
@@ -121,10 +132,36 @@ class OldPlacementGroupResourceManager : public PlacementGroupResourceManager {
   /// creation.
   absl::flat_hash_map<BundleID, std::shared_ptr<BundleState>, pair_hash>
       bundle_state_map_;
+};
 
-  /// Save `BundleSpecification` for cleaning leaked bundles after GCS restart.
-  absl::flat_hash_map<BundleID, std::shared_ptr<BundleSpecification>, pair_hash>
-      bundle_spec_map_;
+/// Associated with new scheduler.
+class NewPlacementGroupResourceManager : public PlacementGroupResourceManager {
```

Comment:
Can you remove New? Let's just call it placement group resource manager

Reply:
em... but the interface has took the name `PlacementGroupResourceManager `,

Reply:
Hmm I see. Maybe we should change the interface to `PlacementGroupResourceManagerInterface`? This class won't be a "New" placement group manager anymore eventually.

Reply:
@raulchen  's suggestion is remove the `interface`, 🤣

Reply:
What about we remove change `NewPlacementGroupResourceManager` to `PlacementGroupResourceManagerImpl` when we remove the old one?

Reply:
That's reasonable to me,